src/js/elements/subindicator_filter/filter_controller.js

```diff
@@ -139,9 +149,24 @@ export class FilterController extends Observable {
         let isExtra = false;
 
         let filterRow = this.addEmptyFilter(isDefault, isExtra);
+        let selectedRandomVal = group.values[Math.floor(Math.random() * group.values.length)];
         filterRow.indicatorDropdown.disable();
-        filterRow.indicatorDropdown.setText(group.name)
+        filterRow.indicatorDropdown.setText(group.name);
+        filterRow.subIndicatorDropdown.setText(selectedRandomVal);
         filterRow.model.currentIndicatorValue = group.name;
+        filterRow.model.currentSubindicatorValue = selectedRandomVal;
+    }
+
+    addDefaultFilter(group) {
+        let isDefault = true;
+        let isExtra = false;
+
+        let filterRow = this.addEmptyFilter(isDefault, isExtra);
+        filterRow.indicatorDropdown.disable();
+        filterRow.indicatorDropdown.setText(group.group);
+        filterRow.subIndicatorDropdown.setText(group.value);
```

Comment:
I don't think this should happen here, rather, you set the `currentIndicatorValue` and `currentSubindicatorValue` in the next two lines, the model will then fire an event indicating that the value has changed, and the dropdown controller would then update itself